tests/rm.bats

```diff
@@ -52,3 +52,13 @@ load helpers
   run_buildah 125 rm -a "$cid"
   expect_output --substring "when using the --all switch, you may not pass any containers names or IDs"
 }
+
+@test "remove a single tagged manifest list" {
+  _prefetch busybox
+  run_buildah manifest create manifestsample
+  run_buildah manifest add manifestsample busybox
+  run_buildah tag manifestsample manifestsample2
+  run_buildah manifest rm manifestsample2
+  # Output should only untag the listed manifest nothing else
+  expect_output "untagged: localhost/manifestsample2:latest"
```

Comment:
And now remove `manifestsample` and make sure that its ID is returned.

Reply:
@vrothberg added change in latest commit.